repo/packages/D/dcos-enterprise-cli/12/package.json

```diff
@@ -3,7 +3,7 @@
   "name": "dcos-enterprise-cli",
   "version": "1.0.8",
   "maintainer": "support@mesosphere.io",
-  "minDcosReleaseVersion": "1.8",
+  "minDcosReleaseVersion": "1.9",
```

Comment:
Aren't we worried that we are changing existing/already released package here? Also - why this change in general?